staging/provisioning/windows/kubeproxystart.ps1

```diff
@@ -9,7 +9,13 @@ if ($Global:ClusterConfiguration.Cni.Name -eq "kubenet") {
 $env:KUBE_NETWORK = $KubeNetwork
 $global:HNSModule = "c:\k\hns.psm1"
 $global:KubeDir = $Global:ClusterConfiguration.Install.Destination
-$global:KubeproxyArgList = @("--v=3", "--proxy-mode=kernelspace", "--hostname-override=$env:computername", "--kubeconfig=$KubeDir\config")
+
+if (-not $Global:ClusterConfiguration.Kubernetes.Kubeproxy.ConfigArgs) {
+    $global:KubeproxyArgList = @()
+} else {
+    $global:KubeproxyArgList = $Global:ClusterConfiguration.Kubernetes.Kubeproxy.ConfigArgs
+}
+$global:KubeproxyArgList += @("--v=3", "--proxy-mode=kernelspace", "--hostname-override=$env:computername", "--kubeconfig=$KubeDir\config")
```

Comment:
If args are specified multiple times does kubeproxy take the first or last instance?
I think for kubelet it will take the last instance but will need to check.

If it does take the last instance maybe we should add the args from the config file after the hardcoded ones so people can change things like verbosity on the fly.

Reply:
Other than this comment /lgtm

Reply:
Good point. The last instance works in kubeproxy. Update the code to append the customized args to the hardcoded ones.